pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+Hi there!
+Thanks for contributing to Swarm!
+
+Before opening a Pull Request or reviewing a Pull Request, please get yourself acquainted with the following guidelines:
+
+1. Open PRs preemptively (i.e. whenever you start working on it), especially for the case of large tasks
+2. Create PRs with a healthy amount of description. A PR without a description will not be approved
+3. PR names should adhere to the following standard: "(affected package(s)): description". examples: "storage: fix panic on hasherstore put", "network, storage, p2p: integrate capabilities api". if many (many) packages are affected, use "all" instead of individual names
+3. Mark the PR with a `ready for review` label once it is. If a second(or more) round of reviews is required, remove the `ready for review` label and add a `ready for another review` label
+4. Never(!) force-push a PR once reviews have started! This has the unfortunate side-effect of resolving/outdating previous comments over the PR
+5. It is up to the reviewer to resolve a comment
+6. A comment does not account as a blocker to merging a PR unless explicitly mentioned in the comment: `BLOCKER: .....`
+7. When reviewing - please make the comment on a line that would constitute a change when addressed - this will resolve the comments automatically once addressed
+8. Once you address a review comment with a code change - no need to comment on it with "done"/"ok"/:+1: - this generates redundant email notifications (to those who have them enabled) and once you push git will automatically resolve/outdate the comment and the reviewer could then mark the comment as resolved without any further taxation of reading even more comments on the PR
+9. We are all here to ship working software with a continuous attention to technical excellence and good design (https://agilemanifesto.org/principles.html); not to make each other's lives miserable. Thus, we ask you to take PR reviews as a process of reflection rather than a fencing duel(!)
```

Comment:
I'd like to add something like "in difference of opinion focus on the arguments." and perhaps "matters of pure taste should be the discretion of thte author"

Reply:
can you please write the complete suggested bulletpoint?